src/mediastreambrokerobserver/MediaStreamBrokerObserver.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * [[MediaStreamBrokerObserver]] can be registered with a [[MediaStreamBroker]] to receive callbacks on media stream
+ * change events.
+ */
+export default interface MediaStreamBrokerObserver {
+  /**
+   * Called when video input stream are changed.
```

Comment:
=> `when the video input stream is changed.`

Update comments for two other methods.